src/jdk.incubator.code/share/classes/jdk/incubator/code/type/ConstructorRef.java

```diff
@@ -42,12 +42,12 @@
 public sealed interface ConstructorRef extends JavaRef, TypeVarRef.Owner
         permits ConstructorRefImpl {
 
+    FunctionType type();
+
     default TypeElement refType() {
```

Comment:
I don't get this default method -- the concrete impl overrides it.

Reply:
I'll clean that up, there is no need for the extra field in `ConstructorRefImpl`.